docs/recipes/add_forcing_input.rst

```diff
@@ -0,0 +1,183 @@
+Include a New Forcing/Input File
+================================
+
+.. use = for sections, ~ for subsections and - for subsubsections
+
+**Feature available since version:** 4.2
+
+There are several ways of including a new forcing or input file into your experiment
+depending on the degree of control you'd like to achieve. An important clarification is
+that ``<forcing/input>_sources`` file dictionary specifies the **sources** (paths to
+the files in the pools or personal folders, that need to be copied or linked into the
+experiment folder). On the other hand ``<forcing/input>_files`` specifies which of these
+sources are to be **included in the experiment**. This allows us to have many sources
+available already for the user and then the user can simply select which of them wants
```

Comment:
This allows us to have many sources available already for the user and then the user can simply select which of them wants to use from ``<forcing/input>_files``.

This allows us to have many sources already available to the user, and the user can then simply choose which of them to use by chosing from ``<forcing/input>_files``.